style.css

```diff
@@ -1,225 +1,658 @@
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-left: 16px;
+  padding: 0;
+  width: 295px;
+  height: 80px;
+}
+
+.frame {
   display: flex;
-  justify-content: space-between;
+  flex-direction: row;
+  gap: 25px;
+  margin-top: -12px;
+  margin-left: 16px;
 }
 
-.hamburger {
-  display: block;
+.list-item {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 13px;
+  line-height: 16px;
 }
 
-.bar {
-  display: block;
-  height: 3px;
-  width: 20px;
-  background-color: #6070ff;
-  margin: 3px;
+.author {
+  list-style-type: none;
+  font-weight: 500;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 0;
 }
 
-.icons > a {
-  padding: 0 0.4em;
+.role,
+.year {
+  list-style-type: disc;
+  font-weight: 400;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 0;
 }
 
-.icons {
-  margin-left: -3em;
+.primary-text2 {
+  position: static;
+  width: 90%;
+  height: auto;
+  left: 0;
+  top: 92px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 16px;
+}
+
+.tags {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  margin-left: 16px;
+  padding: 0;
+  order: 2;
+  flex-wrap: wrap;
 }
 
-.menu a {
+.tag {
+  float: right;
+  text-align: center;
+  background-color: #ebebff;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
   color: #6070ff;
-  font-weight: 700;
-  margin-left: 1rem;
-  line-height: 20px;
-  font-size: 18px;
+  padding: 5px;
+  margin-right: 8px;
+}
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  margin-top: 20px;
+  position: absolute;
+  width: 124px;
+  height: 48px;
+  left: 0;
+  top: calc(50% - 48px / 2);
+  background: #fff;
+  border: 1px solid #7f8cff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  order: 3;
+  margin-left: 24px;
+}
+
+.action {
+  position: static;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  flex-grow: 0;
   text-decoration: none;
+  margin-top: 12px;
+  margin-bottom: 16px;
+  margin-left: 16px;
+}
+
+.btn:hover {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  margin-top: 20px;
+  position: absolute;
+  width: 124px;
+  height: 48px;
+  left: 0;
+  top: calc(50% - 48px / 2);
+  background: #6070ff;
+  border: 1px solid #7f8cff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  order: 3;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  cursor: pointer;
+}
```

Comment:
- [ ] Excellent job adding hover animation in your buttons 🥇, I think it looks awesome, but as requested by the previous reviewer [here](https://github.com/brainskev/setup-and-mobile-version-skeleton/pull/11#discussion_r1014498025), you should stick to Figma design, Kindly edit the enabled, pressed, hover, and disabled properties in all your buttons like this:

![image](https://user-images.githubusercontent.com/91301423/200137526-0ed1ada5-e5c2-4474-ac61-735ad1118d1d.png)